File: python/grains/grains.py
def square(number):
    if number >= 1 and number <= 64:
        return 1 << (number - 1)
    raise ValueError("square must be between 1 and 64")


def total():
    # return sum([1 << n for n in range(64)])
    return (1 << 64) - 1

File: go/grains/grains.go
package grains

import "errors"

func Square(n int) (uint64, error) {
	if n <= 0 || n >= 65 {
		return 0, errors.New("number must be (0-64)")
	}
	return 1 << (n - 1), nil
}

func Total() uint64 {
	// it doesn't use above square function
	return (1 << 64) - 1
}
